apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -401,6 +401,30 @@ def get_enumeration_values(
 
         return self._handle.get_enumeration_values(column_names)
 
+    def extend_enumeration_values(
+        self: DataFrame, values: dict[str, pa.Array], dupes_ok: bool = False
```

Comment:
suggest that the second param be renamed to `deduplicate`, which is the action it controls.  The current name suggests that it will "allow" the final enum to contain duplicate values, which isn't ever possible.